### PR TITLE
[WebcamAPI] Add `BottomUpPoseEstimatorNode` , `MatchStickMenNode`, and `ELkHornNode` and config

### PR DESCRIPTION
**Pull request by liqikai9**
## Modification

Add `BottomUpPoseEstimatorNode` in `tools/webcam/webcam_apis/nodes/mmpose_nodes.py`.

Add `MatchStickMenNode` in `tools/webcam/webcam_apis/nodes/frame_effect_nodes.py`.

Add `ELkHornNode` in `tools/webcam/webcam_apis/nodes/frame_effect_nodes.py`.

Fix bottom up model inference on coco-wholebody dataset in `mmpose/apis/inference.py`.

Add config in `tools/webcam/configs/matchstickmen.py`.

## Checklist

**Before PR**:

- [ ] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) to create this PR.
- [ ] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) are used to fix the potential lint issues.
- [ ] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] New functionalities are covered by complete unit tests. If not, please add more unit tests to ensure correctness.
- [ ] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: tools/webcam/configs/matchstickmen/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# MatchStickMen and ElkHorn Effects
+
+We can apply MatchStickMen and ElkHorn effects on videos with pose estimation results.
```
Comment (ly015) [inline tools/webcam/configs/matchstickmen/README.md:3]: The description is a little too long. We don't need to tell users the details like "modifying skeleton links". Just describe how to play with the demo, like:
- be prepared for the matchstick version of yourself with an elkhorn.
- What's more? Try gesturing a heart with your hands and see what will happen :)

Comment (liqikai9) [reply tools/webcam/configs/matchstickmen/README.md]: Fixed.

**Review by ly015 — APPROVED** [review]

File: tools/webcam/webcam_apis/webcam_runner.py [hunk 2]
```diff
@@ -108,7 +108,7 @@ def __init__(self,
 
         # Register runner for all nodes
         for node in self.node_list:
-            logging.info(f'Set runner for node: {node.name})')
+            logging.info(f'Set runner for node: ({node.name})')
```
Comment (ly015) [inline tools/webcam/webcam_apis/webcam_runner.py:111]: Better just remove the brackets.

File: mmpose/apis/inference.py [hunk 3]
```diff
@@ -565,7 +565,11 @@ def inference_bottom_up_pose_model(model,
             })
 
         # pose nms
-        keep = oks_nms(pose_results, pose_nms_thr, sigmas=None)
```
Comment (ly015) [inline mmpose/apis/inference.py:568]: Please rebase to the latest webcam_api branch, which already handles the sigma issue

**Pull request closed without merging**